src/screens/OtherProfile/index.js

```diff
@@ -642,9 +637,12 @@ const OtherProfile = () => {
           extraData={[feeds]}
           data={handleDataFeed()}
           onScroll={handleScroll}
-          onEndReach={__handleOnEndReached}
           onRefresh={handleRefresh}
           refreshing={loading}
+          onMomentumScrollEnd={(event) => {
+            onWillSendViewPostTime(event, feeds);
+            fetchNextFeeds(event);
+          }}
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The `onEndReach` prop has been removed and replaced with `onMomentumScrollEnd`. This change seems to be part of the shift from using `isLastPage` to determine when to fetch more feeds to using the `fetchNextFeeds` function from the `useFeedPreloadHook`.

However, as mentioned earlier, it's important to ensure that the `postOffset` is being correctly updated when fetching the next set of feeds. If `postOffset` isn't properly updated, this could lead to issues with fetching and displaying the correct feeds.

Please ensure that the `postOffset` is being correctly updated when fetching the next set of feeds.


<!-- This is an auto-generated comment by OSS CodeRabbit -->